src/PhpSpreadsheet/Calculation/Calculation.php

```diff
@@ -754,17 +754,17 @@ class Calculation
         ],
         'DATE' => [
             'category' => Category::CATEGORY_DATE_AND_TIME,
-            'functionCall' => [DateTime::class, 'DATE'],
+            'functionCall' => [DateTimeExcel\Datefunc::class, 'funcDate'],
             'argumentCount' => '3',
         ],
         'DATEDIF' => [
             'category' => Category::CATEGORY_DATE_AND_TIME,
-            'functionCall' => [DateTime::class, 'DATEDIF'],
+            'functionCall' => [DateTimeExcel\Datedif::class, 'funcDatedif'],
```

Comment:
Perhaps use `DateDif` rather than `Datedif`: the capitalisation makes it easier to read at a glance.

Reply:
Harder than it seems, because I'm developing on Windows with non-case-sensitive file names. Anticipating this difficulty, I tried several pushes till I found a scheme that works - rename to something entirely different, push, rename to desired name, push. So far, only DateDif is changed; I will work on the others later today.

Reply:
I'm on Windows myself (using Linux vagrant boxes for running the tests), so I've had that pain as well... and that's the only workround that I've ever found for the case-sensitive renames